src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -46,7 +46,22 @@ public class ValueMappingEstimator : TrivialEstimator<ValueMappingTransformer>
         /// <param name="columns">The list of names of the input columns to apply the transformation, and the name of the resulting column.</param>
         internal ValueMappingEstimator(IHostEnvironment env, IDataView lookupMap, string keyColumn, string valueColumn, params (string outputColumnName, string inputColumnName)[] columns)
```

Comment:
>ValueMappingEstimator [](start = 17, length = 21)

I'm not getting why we still have this constructor
Public interface operates over Schema.Column object, any conversion from string to Schema.Column is unrelaable, why it's still around?

Reply:
Removed and related code is updated as well.

---
In reply to: [271544302](https://github.com/dotnet/machinelearning/pull/3180#discussion_r271544302) [](ancestors = 271544302)